library/build.gradle

```diff
@@ -1,6 +1,14 @@
 apply plugin: 'com.android.library'
 apply from: '../maven_push.gradle'
 
+// https://docs.gradle.org/current/userguide/publishing_maven.html
+// http://www.flexlabs.org/2013/06/using-local-aar-android-library-packages-in-gradle-builds
+apply plugin: 'maven-publish'
```

Comment:
@bpappin using maven-publish plugin, we can remove maven_push.gradle right?

cc @satyan